src/webparts/reactMyGroups/components/reactMyGroups/ReactMyGroups.tsx

```diff
@@ -48,27 +54,41 @@ export class ReactMyGroups extends React.Component<IReactMyGroupsProps, IReactMy
 
     return (
       <div className={styles.reactMyGroups} style={{ backgroundColor: semanticColors.bodyBackground }}>
-        <div className={styles.title} role="heading" aria-level={2}>{(strings.userLang == "FR" ? this.props.titleFr :this.props.titleEn )} </div>      
+        <div className={styles.title} role="heading" aria-level={2}>{(strings.userLang == "FR" ? this.props.titleFr :this.props.titleEn )} </div> 
+        <div className={styles.seeAll}>{this.props.toggleSeeAll == false && <a href={this.props.seeAllLink}>{strings.seeAll}</a>}</div>
+        <div className={styles.createComm}><Icon iconName="Add" className={styles.addIcon} /><a href={this.props.createCommLink}>{strings.createComm}</a></div>  
           {this.state.isLoading ?
-            <Spinner label="Loading sites..." />
-                : 
-                <div>
-                  {this.props.layout == 'Compact' ?
-                    <GroupList groups={pagedItems} onRenderItem={(item: any, index: number) => this._onRenderItem(item, index)}/>
-                  :
-                    <GridLayout sort={this.props.sort} items={pagedItems} onRenderGridItem={(item: any, finalSize: ISize, isCompact: boolean) => this._onRenderGridItem(item, finalSize, isCompact)}/>
-                  }
+    <Spinner label="Loading sites..." />
```

Comment:
Everything looks good! Just a quick change, this label will need a translation string.